src/beachbot/manipulators/roarmm1.py

```diff
@@ -170,6 +170,8 @@ def refresh_robot_state(self):
                     # robot status package recieved:
                     qs = [float(data.get("A" + str(num + 1), 0)) for num in range(5)]
                     taus = [float(data.get("T" + str(num + 1), 0)) for num in range(5)]
+                    # Convert gripper joint angle into range 0..1
```

Comment:
@jeffreyfq  did you make a similar change in the simulator arm? I'm not seeing it. Won't these be two different interfaces then?